example/submit.pbs

```diff
@@ -1,25 +1,44 @@
-#!/bin/sh
+#!/bin/bash
+#PBS -l nodes=1:ppn=5
+#PBS -N qespresso_NAC
+#PBS -j oe
+#PBS -q batch
+#PBS -l walltime=30:00:00
 
-MPI='mpirun'
-NP=20
-VASP='/opt/vasp/vasp.5.3/vasp'
+cd $PBS_O_WORKDIR
+echo $PBS_O_WORKDIR
 
-exe_qespresso=pw.x
-exe_export=pw_export.x
-exe_convert=iotk
+###### some conventional settings ###########
+export PYTHONPATH=/home/eric/src/Libra/libra-code/_build/src:$PYTHONPATH
+export LD_LIBRARY_PATH=/home/eric/src/Libra/libra-code/_build/src:$LD_LIBRARY_PATH
+export PYTHONPATH=/home/eric/src/Libra/pyxaid2:$PYTHONPATH
+export LD_LIBRARY_PATH=/home/eric/src/Libra/pyxaid2:/opt/boost/1.55.0/lib:$LD_LIBRARY_PATH
+export LD_LIBRARY_PATH=/opt/intel/composer_xe_2013.1.117/mkl/lib/intel64:$LD_LIBRARY_PATH
+
+#### path for the QE module ####
+exe_qespresso=/home/eric/src/qe/espresso-5.1.2/bin/pw.x
+exe_export=/home/eric/src/qe/espresso-5.1.2/bin/pw_export.x
+exe_convert=/home/eric/src/qe/espresso-5.1.2/bin/iotk
+
+### MPI ###
+MPIRUN=/home/eric/src/ompi/1.8.5-gcc/bin/mpirun
```

Comment:
please add a comment - this won't work with all environments, e.g. my slurm requires "srun"